dashboard/app/models/census/state_cs_offering.rb

```diff
@@ -18,15 +18,17 @@ class Census::StateCsOffering < ApplicationRecord
   belongs_to :school, foreign_key: :state_school_id, primary_key: :state_school_id, required: true
 
   validates_presence_of :course
-  validates :school_year, presence: true, numericality: {greater_than_or_equal_to: 2016, less_than_or_equal_to: 2030}
+  validates :school_year, presence: true, numericality: {greater_than_or_equal_to: 2015, less_than_or_equal_to: 2030}
 
   SUPPORTED_STATES = %w(
     AR
     CA
+    CT
```

Comment:
Two states included deliberately?

Reply:
Yes.  Git has been having trouble merging these state course import branches when there are multiple open at the same time, I believe because it does not know how to line up the case conditions.  So, I've been branching from the prior state branch, and then merging them into staging in the order that I created the branches.